src/Appium.Net/Appium/Windows/WindowsDriver.cs

```diff
@@ -110,6 +111,31 @@ public WindowsDriver(Uri remoteAddress, AppiumOptions AppiumOptions, TimeSpan co
         {
         }
 
+        #region Contex
+
+        public override string Context
+        {
+            get
+            {
+                return "NATIVE_APP";
```

Comment:
- Does windows driver have no implementation for `get context` as well?
- Does windows driver raise `InvalidOperationException` for `set`? If so, we simply though the error to users.
    - We can reduce client-side implementation

Reply:
WinAppDriver returns Unknown Command as it can be seen here https://gist.github.com/cosminstirbu/14375a1f101ad5fed16648a37b746102

On the client side client, the exception is thrown in `ContentMappedBy` via `GetReturnRelevantBys`. I could do a try / catch there on the catch clause `return map[ContentTypes.NATIVE];`

Reply:
However this would "mess up" the logs since this call will be frequently logged :-?

Reply:
> However this would "mess up" the logs since this call will be frequently logged :-?

It's trade-off thing, I would say.
We are implementing clients to throw errors to users such as `not implemented yet` without modification in clients layer as possible. We can fix the error only updating Appium server side. We do not need to force users to update frequently.

I am not familiar with windriver's situation. Thus, I'm fine to go this way 👍if it is better for windows driver.